src/app/workspace/resource/properties/properties.component.scss

```diff
@@ -166,6 +166,15 @@
   color: rgba(0, 0, 0, 0.54);
 }
 
+:host::ng-deep .mat-paginator-container {
```

Comment:
`ng-deep` is deprecated. It would be a better way to define/overwrite component's style in one of the global stylesheets in the assets folder. But for the moment you can keep it as it is.

Reply:
Ok, I keep this for the moment. Is there an alternative?

Reply:
There's no alternative for `ng-deep` and it doesn't appear as if there ever will be an alternative so you'll have to overwrite the component's style either in one of the global style sheets (recommended) or directly in the stylesheet of the component